brainstormforce/ultimate-addons-for-gutenberg
# New: WP 5.7 Compatibility

### Description

Trello - https://trello.com/c/raylEFaO/197-wp-57-testing

### Screenshots


### Types of changes




- Bug fix

### How has this been tested?

- Drag and drop each module.
- Check it's not breaking anything.
- Drag and drop every block with no console error in the editor.

### Checklist:
